R/ds.studentTestDP.R

```diff
@@ -58,7 +60,7 @@ computeGlobalStats <- function(Nstudies, input) {
 computeStudentTestCombine <- function(Nstudies, input_x, input_y) {
     x_stats <- computeGlobalStats(Nstudies, input_x) 
     y_stats <- computeGlobalStats(Nstudies, input_y)
-    StudentTest <- (x_stats$GlobalMean - y_stats$GlobalMean)/sqrt(x_stats$GlobalVar / x_stats$GlobalNvalid + y_stats$GlobalVar / y_stats$GlobalNvalid)
+    StudentTest <- (x_stats$GlobalMean - y_stats$GlobalMean)/sqrt(max(x_stats$GlobalVar / x_stats$GlobalNvalid + y_stats$GlobalVar / y_stats$GlobalNvalid, delta))
```

Comment:
I think I would have put the max on the var computation which is - I think - equivalent and makes it clear that the variance is a positive number